pyaptamer/datasets/_loaders/_online_databank.py

```diff
@@ -0,0 +1,27 @@
+from Bio.PDB import PDBList
+
+from pyaptamer.utils.pdb_to_struct import pdb_to_struct
+from pyaptamer.utils.struct_to_aaseq import struct_to_aaseq
+
+
+def download_and_extract_sequences(pdb_id):
+    """
```

Comment:
this docstring is incomplete. Where is it downloading from, is it creating files on the hard drive, if yes where, etc

Also, why do we need this composition?

I would rmeove `struct_to_aaseq`, since only the AptaNet algorithm needs this.

I would design it without, and also enable caching - i.e., if the file already exists, and `force_download=False`, it does not get overwritten.